### PR TITLE
Update the README with the multiple certificates creation information

### PR DESCRIPTION
**Pull request by danypr92**
Solves the issue #11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,10 @@ This role will:
 1. Add `certbot` PPA repository
 2. Install `certbot` and `python-certbot-nginx` packages
 3. `certbot` package will add a `renew` cron job and a systemd-timer ([More info](https://certbot.eff.org/#ubuntuxenial-nginx)
-4. Generate a Let's Encrypt SSL certificate for the given `domain_name`
+4. Generate a Let's Encrypt SSL certificates for the given `domain_name` or `domain_names`.
```
Comment (enricostano) [inline README.md]: This is not true, the `domain_names` thing it's external to this role. That variable is not used inside this role. This role only use `domain_name` variable.

File: README.md [hunk 2]
```diff
@@ -7,7 +7,10 @@ This role will:
...
+
+Also allow to generate and manage multiple certificates in the same host.
+You can define the var `domain_names` and configure a role to incude the this role in a loop. See [Multiple certificates creation](#example-playbook---multiple-certificates-creation) section.
```
Comment (enricostano) [inline README.md]: I would create a new paragraph like *Generating multiple certificates* and keep all the things (examples, explication, etc.) related to multiple domains in there.

What do you think? The idea is to explain first the easy case and then the more complex case.

**Review by enricostano — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -19,19 +22,52 @@ certbot_nginx_cert_name: mycert # optional
...
 ```
 
+Example Playbook - Multiple certificates creation
+-----------------------------------------
+
+```yaml
+# Playbook
+- hosts: servers
+  roles:
+    - role: coopdevs.certbot-nginx
```
Comment (enricostano) [inline README.md]: `certbot_nginx`

**Comment by danypr92** [comment]
I apply all the changes requested @enricostano. If you don't have any request, we can merge it!

**Review by enricostano — APPROVED** [review]

**Pull request merged**